list.rb

```diff
@@ -0,0 +1,29 @@
+require_relative 'enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
```

Comment:
- [ ] `MyList` class has the `#initialize` method that accepts multiple arguments. I have showed you how you can do that by using your implemented `#each` method.

```ruby
def initialize(*args)
        @list = []
        args.each { |arg| @list << arg }
    end
```

Currently, I am getting this answer after running this command.

![Screenshot from 2022-01-21 21-34-51](https://user-images.githubusercontent.com/22434660/150560390-529e8521-158f-4c0a-a5e9-a2e907c736ba.png)